torch/csrc/cudnn/Descriptors.h

```diff
@@ -29,6 +29,18 @@ inline cudnnDataType_t getDataType(const at::Tensor& t) {
   throw std::runtime_error("TensorDescriptor only supports double, float and half tensors");
 }
 
+static inline void fixSizeOneDimStride(int dim, int *size, int *stride) {
```

Comment:
For future reference, this is the kind of function which really deserves a comment. (My rewrite of cudnn library will add the comment so you don't have to do anything here.)